applications/tari_console_wallet/src/ui/components/transactions_tab.rs

```diff
@@ -331,6 +336,19 @@ impl TransactionsTab {
                 tx.transaction.body.kernels()[0].excess_sig.get_signature().to_hex()
             };
             let excess = Span::styled(excess_hex.as_str(), Style::default().fg(Color::White));
+            let confirmation_count = app_state.get_confirmations().get(&tx.tx_id);
```

Comment:
minor: this would be a bit nicer as `app_state.get_confirmations(&tx.tx_id)` I think

Reply:
Will fix

Reply:
Done.